pkg/lib/infra/redis/hub.go

```diff
@@ -29,14 +24,32 @@ func (i *SubscriberID) Next() int64 {
 	}
 }
 
-type ChannelKey struct {
-	ConnKey     string
-	ChannelName string
+type Subscriber struct {
+	ChannelName  string
+	SubscriberID int64
+	// The drop pattern
+	// See https://www.ardanlabs.com/blog/2017/10/the-behavior-of-channels.html
+	MessageChannel chan *redis.Message
 }
 
-type Subscriber struct {
+// ControlMessageJoin is essentially Subscriber.
+type ControlMessageJoin = Subscriber
```

Comment:
This is a type alias instead of type definition, which should be seldom used. Can we just do `type ControlMessageJoin Subscriber` instead, and cast `ControlMessageJoin` to `Subscriber` in relevant places?

Reply:
I changed it to type definition and ignored the lint error.